src/Codeception/Test/Unit.php

```diff
@@ -93,15 +97,12 @@ public function getModule($module)
     /**
      * Returns current values
      */
-    public function getCurrent($current)
+    public function getCurrent(?string $current): ?array
```

Comment:
This `getCurrent` returns `mixed` too.
The signature must mach `getCurrent` in `Metadata` class.